app/controllers/npq_separation/admin/cohorts_controller.rb

```diff
@@ -0,0 +1,78 @@
+class NpqSeparation::Admin::CohortsController < NpqSeparation::AdminController
+  before_action :ensure_super_admin, except: %i[index show]
+  before_action :ensure_editable, only: %i[edit update destroy]
+
+  def index
+    @pagy, @cohorts = pagy(Cohort.all.order(start_year: :desc))
+  end
+
+  def show
+    @cohort = cohort
+  end
+
+  def new
+    @cohort = Cohort.new
+    render :form
+  end
+
+  def create
+    @cohort = Cohort.new(cohort_params)
+
+    if @cohort.save
+      flash[:success] = "Cohort created"
+      redirect_to action: :index
+    else
+      render :form, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @cohort = cohort
```

Comment:
```suggestion
    cohort
```
the assignment to `@cohort` is already done in `def cohort`